### ISSUE TITLE
indentation in nested Sass Maps

### ISSUE THREAD

**Issue by alan-agius4** [stylelint/stylelint#1406]
"A bug where the indentation rule warning is being firing when it is not suppose to in nested sass maps"

> Which rule, if any, is this issue related to?

e.g. `indentation`

> What CSS is needed to reproduce this issue?

e.g.

```css
$colors: (
	primary: (
		base: $rouge,
		contrast: $white
	)
);
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
"indentation": [
	"tab",
	{
		"indentClosingBrace": false,
		"indentInsideParens": "once" 
	}
],
}
```

> Which version of stylelint are you using?

`6.5.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

"gulp"

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

"Yes, it's related to SCSS maps.

> What did you expect to happen?

"No warnings to be flagged."

> What actually happened (e.g. what warnings or errors you are getting)?

"The following warnings were flagged:"

```shell
Expected indentation of 1 tab (indentation
56:1    ‼  Expected indentation of 1 tab (indentation)
```
